python/cuml/common/base.pyx

```diff
@@ -19,68 +19,123 @@
 # cython: embedsignature = True
 # cython: language_level = 3
 
-
+import cuml
 import cuml.common.handle
 import cuml.common.cuda
 import inspect
 
+from cudf.core import Series, DataFrame
+from cuml.common.array import Array
+from cupy import ndarray as cupyarray
```

Comment:
I would probably prefer `cupy.array`, but I know I'm alone there, so how about `cupyArray` or `cupy_array`? Elsewhere, we don't really use the squashed-names-no-caps convention